lib/topological_inventory/ansible_tower/operations/worker.rb

```diff
@@ -71,6 +73,18 @@ def default_messaging_opts
             :group_ref  => "topological-inventory-operations-ansible-tower"
           }
         end
+
+        # TODO: Probably move this to common eventually, if we need it elsewhere.
+        def heartbeat_thread(client = nil)
+          @heartbeat_thread ||= Thread.new do
+            loop do
+              sleep(15)
+              client.send(:topic_consumer, queue_opts[:persist_ref], nil).trigger_heartbeat!
+            rescue => e
```

Comment:
Is there a real issue with this placement? Rubocop seems to think so.

Reply:
I'm thinking it's a ruby version thing - thats ruby 2.4, I can add `rescue StandardError => e` and then dippy-bot will complain too. Ill just make it explicit to make hakiri happy.

Reply:
it would be best to remove this rule at all, because `rubocop -a` is causing syntax error, right?